newIDE/app/src/ResourcesList/ResourceSelector.js

```diff
@@ -42,6 +48,10 @@ const styles = {
 };
 
 export default class ResourceSelector extends React.Component<Props, State> {
+  static defaultProps = {
```

Comment:
Avoid defaultProps, they make things more complicated from my experience. If the prop is not passed, it will be undefined.